SmartDeviceLink/SDLFileManagerConstants.h

```diff
@@ -10,8 +10,17 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
+typedef NSString SDLFileName;
+
 typedef void (^SDLFileManagerUploadCompletionHandler)(BOOL success, NSUInteger bytesAvailable, NSError *__nullable error);
+typedef void(^SDLFileManagerMultiUploadCompletionHandler)(NSError *__nullable error);
+typedef void(^SDLFileManagerMultiUploadProgressHandler)(SDLFileName *fileName, float uploadPercentage, BOOL *cancel, NSError *__nullable error);
```

Comment:
The progress handler should be altered to match the [RPC sequential send proposal design](https://github.com/joeljfischer/sdl_evolution/blob/2b41868dc7a666ab498dd5643a2e068bb5b96134/proposals/nnnn-send-multiple-rpcs-ios.md#proposed-solution) due to the current design's difficulty of compatibility with Swift.

Reply:
Changed. Removed `cancel` as a parameter. The remaining uploads are now only canceled when NO is returned in the progress handler.